src/runtime/pkg/sev/sev.go

```diff
@@ -125,7 +139,8 @@ func constructSevHashesTable(kernelPath, initrdPath, cmdline string) ([]byte, er
 }
 
 // CalculateLaunchDigest returns the sha256 encoded SEV launch digest based off
-// the current firmware, kernel, initrd, and the kernel cmdline
+// the current firmware, kernel, initrd, and the kernel cmdline. initrdPath and
+// cmdline may be empty.
```

Comment:
Can't the kernel path also be empty? That will just skip calculating the hash table completely, which is probably a more common use case than using kernel hashes without the initrd or cmdline. I guess there is a comment that explains it below, but maybe should be in the README also?

Reply:
A closer look shows that we already support measurement without kernel hashes:

https://github.com/kata-containers/kata-containers/blob/a2e69c5b6654f34cb28734ced2bad847f051801a/src/runtime/pkg/sev/sev.go#L141-L151

But the function documentation doesn't state it clearly; I'll improve that.

Reply:
Update function doc.